api/src/test/java/org/openmrs/module/openconceptlab/updater/ImporterTest.java

```diff
@@ -671,6 +716,77 @@ public void importMapping_addConceptMappingAndUnretireTerm() throws Exception {
 		ConceptMapType mapType = conceptService.getConceptMapTypeByName("SAME_AS");
 		assertThat(concept.getConceptMappings(), contains(hasMapping(source, "1001", mapType)));
 	}
+	
+	@Test
+	public void importMapping_shouldUpdateMappingOnylIfItHasBeenUpdatedSinceLastImport() throws Exception {
+		OclConcept oclConcept = newOclConcept();
+		updateService.saveItem(importer.importConcept(new CacheService(conceptService), update, oclConcept));
+		Update update = updateService.getLastUpdate();
+
+		OclMapping oclMapping = new OclMapping();
+		oclMapping.setExternalId("dde0d8cb-b44b-4901-90e6-e5066488814f");
+		oclMapping.setMapType("SAME-AS");
+		oclMapping.setFromConceptUrl("/orgs/CIELTEST/sources/CIELTEST/concepts/1001/");
+		oclMapping.setToSourceName("SNOMED CT");
+		oclMapping.setToConceptCode("1001");
+		oclMapping.setUrl("/orgs/CIELTEST/sources/CIELTEST/mappings/303");
+
+		updateService.saveItem(importer.importMapping(new CacheService(conceptService), update, oclMapping));
+		
+		oclMapping.setUpdatedOn("2008-02-18T09:10:16Z");
+		
+		Item item = importer.importMapping(new CacheService(conceptService), update, oclMapping);
+		assertThat(item, hasProperty("state", equalTo(ItemState.UPDATED)));
+		updateService.saveItem(item);
+		
+		item = importer.importMapping(new CacheService(conceptService), update, oclMapping);
+		assertThat(item, hasProperty("state", equalTo(ItemState.ALREADY_UP_TO_DATE)));
+	}
+	
+	@Test
+	public void isMappingUpdatedSince_shouldReturnIfMappingUpdateOnIsAfter() throws Exception{

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/agrzybkowski/openmrs-module-openconceptlab/file/2377174879/issues/source?bid=3218637&fileBranchId=3224807#l747)